From b5c81e276bb5c8e0a0da97bd2e21db73428b9701 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 12:58:42 -0400
Subject: [ethernet] qlge: Get rid of an redundant assignment

Message-id: <1408625927-3701-12-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88528
O-Subject: [PATCH RHEL 7.1 11/16] qlge: Get rid of an redundant assignment
Bugzilla: 1089357
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Alexander Gordeev <agordeev@redhat.com>

Bug 1089357 (https://bugzilla.redhat.com/show_bug.cgi?id=1089357)

net commit id 7f8358c908c1c3bc3c222ebc1ea5dba8c8dc0c1a

Signed-off-by: Alexander Gordeev <agordeev@redhat.com>
Cc: Shahed Shaikh <shahed.shaikh@qlogic.com>
Cc: Jitendra Kalsaria <jitendra.kalsaria@qlogic.com>
Cc: Ron Mercer <ron.mercer@qlogic.com>
Cc: linux-driver@qlogic.com
Cc: netdev@vger.kernel.org
Cc: linux-pci@vger.kernel.org
Acked-by: Jitendra Kalsaria <jitendra.kalsaria@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlge/qlge_main.c b/drivers/net/ethernet/qlogic/qlge/qlge_main.c
index 5314dfe..777fab3 100644
--- a/drivers/net/ethernet/qlogic/qlge/qlge_main.c
+++ b/drivers/net/ethernet/qlogic/qlge/qlge_main.c
@@ -3348,7 +3348,6 @@ static void ql_enable_msix(struct ql_adapter *qdev)
    qdev->msi_x_entry = NULL;
    netif_warn(qdev, ifup, qdev->ndev,
        "MSI-X Enable failed, trying MSI.\n");
-   qdev->intr_count = 1;
    qlge_irq_type = MSI_IRQ;
   } else if (err == 0) {
    set_bit(QL_MSIX_ENABLED, &qdev->flags);
-- 
1.7.1